index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Prangon's Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <div class="container">
+            <div class="logo-placeholder">
+                <div class="my-logo">
+                    PG
+                </div>
+            </div>
+            <div class="menu">
+                <div class="icon">
+                    <div class="union">
+                        <button class="nav-button"><img src="images/Union.png" alt="menu-image" class="menu-image"></button>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </header>
+    <div class="container">
+        <div class="headline">
+            <div class="primary-text">
+                I'm Prangon<br>Glad to see you!
+            </div>
+            <div class="description">
+                I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </div>
+            <div class="social-media">
+                LET'S CONNECT<br>
+                <!-- <div class="social-media-icons"> -->
+                    <a href="#"><img src="images/Twitter.png" alt="Twitter" id="twitter"></a>
+                    <a href="#"><img src="images/Linkedin.png" alt="Linkedin" id="linkedin"></a>
+                    <a href="#"><img src="images/M.png" alt="M" id="M"></a>
+                    <a href="#"><img src="images/GitHub.png" alt="GitHub" id="github"></a>
+                    <a href="#"><img src="images/Victory.png" alt="Victory" id="victory"></a>
```

Comment:
- I would like to kindly suggest that you make this an unordered list `<ul>` and `<li>` because these are a list of icons representing each social media logo.

You may as well refactor to look like this 👇🏽 
```
     <ul>
        <li><a href="#"><img src="images/Twitter.png" alt="Twitter" id="twitter"></a></li>
       <li><a href="#"><img src="images/Linkedin.png" alt="Linkedin" id="linkedin"></a></li>
        <li><a href="#"><img src="images/M.png" alt="M" id="M"></a></li>
        <li><a href="#"><img src="images/GitHub.png" alt="GitHub" id="github"></a></li>
         <li><a href="#"><img src="images/Victory.png" alt="Victory" id="victory"></a></li>
       </ul>
```